zkvm/src/constraints.rs

```diff
@@ -12,6 +12,7 @@ use subtle::{ConditionallySelectable, ConstantTimeEq};
 use crate::encoding::*;
 use crate::errors::VMError;
 use crate::scalar_witness::ScalarWitness;
+use readerwriter::Encodable;
```

Comment:
Since we do `use crate::encoding::*` in order to get all the traits we need, why not put `pub use readerwriter::Encodable` in that module, so in all other places we don't have to add `use readerwriter::Encodable`? See also another suggestion in `src/encoding.rs`.

```suggestion
```

Reply:
Ok.